main.js

```diff
@@ -0,0 +1,43 @@
+// Define a function maxOfTwoNumbers that takes 
+//two numbers as arguments and returns the 
+//largest of them. Use the if-then-else construct available in Javascript.
+
+// Define a function maxOfThree that takes three numbers as arguments and returns the largest of them. 
+
+function maxOfTwoNumbers (bigNum, smallNum) {
+    if((bigNum>smallNum)
+  {
+    console.log("This is the largest number");
```

Comment:
this message is kinda vague how about printing the number so it can make sense like ` console.log( bigNum + " This is the largest number");`